### DISCUSSION TITLE
About dragging outside the chart area

### DISCUSSION THREAD

**Discussion by shochu-hiracchi** [chrispahm/chartjs-plugin-dragdata#104] [Q&A]
Precise dragging is required when moving points within a radar chart.

If you pass from the center point, the point will stretch in the opposite direction, and so on.

What I want to do is in the second table.
Dragging only operates within the axis and has no effect on other axes. (It does not extend to the other side even if it passes the center.

https://user-images.githubusercontent.com/114906039/193757162-e47ccf2d-03ae-4729-838a-e7c0d24638ee.mp4

https://user-images.githubusercontent.com/114906039/193757166-b75a0b79-200f-42e0-a342-3927ab9a2b5f.mp4

**Comment by artus9033** [comment]
This indeed was an issue (#105), however was fixed in #120, which was published in version `2.3.0`. Therefore, versions `>= 2.3.0` are free of this problem.
